### PR TITLE
Document url properties

### LINKED ISSUE #1732: Create documenation for elemetn call urls

### PR DESCRIPTION
**Pull request by toger5**
As discussed we want to have documenation of the url parameters inside this repo.
Fixes: https://github.com/vector-im/element-call/issues/1732

### CODE REVIEW

File: docs/url-params.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+```
+lang: string | null;
+```
+
+**fonts**
+The fonts which the interface should use, if not empty.
+
+```
+fonts: string[];
+```
```
Comment (robintown) [inline docs/url-params.md]: Note that this parameter's value is defined by a `?font=font-one&font=font-two...` in the URL - the name is different, not sure how you want to handle this in the docs

File: docs/url-params.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+confineToRoom: boolean;
+```
+
+**appPrompt**
+Whether upon entering a room, the user should be prompted to launch the
+native mobile app. (Affects only Android and iOS.)
+
+```
+appPrompt: boolean;
+```
```
Comment (robintown) [inline docs/url-params.md]: A lot of these params (for example this one) have default values that would be good to document

**Review by robintown — COMMENTED** [review]
Thanks for doing this! I really like that you've documented the general URL format in detail

File: docs/url-params.md (hunk 1 above)
Comment (toger5) [reply docs/url-params.md]: oh...
I will have a look. Very important to document it then ;)

**Review by robintown — APPROVED** [review]

**Pull request merged**